bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -0,0 +1,234 @@
+            {{ _('Create a Firefox Account') }}
+          </a>
+        </div>
+        <div class="show-fxa-supported-signed-in">
+          <button class="mzp-c-button" type="button" id="fx-modal">{{ _('Download Firefox App') }}</button>
+        </div>
+        <div id="features-hero-download" class="show-fxa-not-fx">
+          {{ download_firefox(dom_id=download_id, download_location=button_location) }}
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="mzp-c-hero-image">
+    {{ high_res_img('firefox/accounts/fxa-hero-img.png', {'alt': '', 'width': '1200', 'height': '770'}) }}
+  </div>
+</section>
+
+<div class="mzp-l-content">
+  <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.') }}</h2>
+</div>
+
+<div class="mzp-l-content">
+  {% call feature_card(
+    title=_('Send tabs instantly to your devices'),
+    image_url='firefox/accounts/fxa-send.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-right-half'
+  ) %}
+  <p>{{ _('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.') }}</p>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Remember your passwords in Lockbox'),
+    image_url='firefox/accounts/fxa-lockbox.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-left-half lockbox-card',
+    link_cta=_('Get the Lockbox App'),
+    link_url='#'
+  ) %}
+    <p>{{ _('No more getting locked out of apps and websites. Lockbox secures all the passwords you’ve saved in Firefox and gives you easy access across all your iOS devices.') }}</p>
+
+    <ul class="mobile-download-buttons mobile-lockbox">
+      <li class="ios">
+        <a id="appStoreLink" href="https://app.adjust.com/kq8zhhz" data-link-type="download" data-download-os="iOS">
+          <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+        </a>
+      </li>
+    </ul>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Sync your bookmarks everywhere'),
+    image_url='firefox/accounts/fxa-sync.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-right-half bookmarks-card'
+  ) %}
+    <p>{{ _('Bookmark a recipe at home. Check it at the market from your phone. Every bookmark you save is instantly added to all your synced devices.') }}</p>
+  {% endcall %}
+
+  {% call feature_card(
+    title=_('Put quality content in your Pocket'),
+    image_url='firefox/accounts/fxa-pocket.png',
+    include_highres_image=True,
+    aspect_ratio='mzp-has-aspect-1-1',
+    class='mzp-l-card-feature-left-half pocket-card',
+    link_cta=_('Get the Pocket App'),
+    link_url='#'
```

Comment:
Missing URL

Reply:
Hm. Normally we'd use a button for a pure JavaScript trigger like that. I'm not sure the macro will support that.  What do you think of pointing to https://getpocket.com/ for users with JS disabled?

Reply:
I wrote some javascript applying the modal click event listener to all card cta links. If I include a link url it will take all users to the address before the modal loads regardless of whether JS is disabled or not. Unsure how to reconcile. 🤔

Reply:
[`e.preventDefault()`](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)!

Reply:
do you think I should link to the app store for the others? Not all of them have an 'official website'

Reply:
Ok. Notes currently doesn't have an adjust link yet so I'll use `https://play.google.com/store/apps/details?id=org.mozilla.testpilot.notes`

Reply:
A bug(?)-- on click, mobile buttons appear behind modal. They disappear again when you close the modal. :( 

<img width="959" alt="screen shot 2018-11-29 at 3 59 04 pm" src="https://user-images.githubusercontent.com/30483988/49259959-429baf80-f3f0-11e8-9a04-5356250917e8.png">

Reply:
That makes sense given what I saw of the javascript.

I wouldn't block launch on it but you can fix it if you like.